packages/mediacenter/kodi/package.mk

```diff
@@ -13,8 +13,8 @@ PKG_PATCH_DIRS="$KODI_VENDOR"
 
 case $KODI_VENDOR in
   raspberrypi)
-    PKG_VERSION="5f134adfcee6fd031c7b8f905bce892dc6d88f7a" # kodi19-pre-Python3
-    PKG_SHA256="9acec0b04f802b8bcd24922a1664ee47198f384b6ba8f0704718dde176a81858"
+    PKG_VERSION="b4e54bf036c7472e6842d5c960a6fdae36ba19f2" # kodi19-pre-Python3
```

Comment:
not sure we want an 3 year old version ? :)

Reply:
Sorry - should be fixed now, thanks for noticing! :)